jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/extensions/compress/CompressExtension.java

```diff
@@ -202,26 +203,22 @@ protected void decompress(ByteAccumulator accumulator, ByteBuffer buf) throws Da
             {
                 if (read == 0)
                 {
-                    LOG.debug("Decompress: read 0 {}", toDetail(inflater));
+                    if (LOG.isDebugEnabled())
+                        LOG.debug("Decompress: read 0 {}", toDetail(inflater));
```

Comment:
This one was important, as the `toDetail()` causes a lot of unnecessary behavior.